includes/news/v2/mail/announcements.php

```diff
@@ -38,8 +38,8 @@
 	<tr>
 		<td style="text-align: right; padding-bottom: 0; font-family: Helvetica, Arial, sans-serif; font-weight: bold; text-transform: uppercase;" align="right">
 			<a href="<?php echo ANNOUNCEMENTS_MORE_URL; ?>">
-				More Announcements
-			</a>
+				More Announcements</a>
+			<img src="<?php echo bloginfo( 'stylesheet_directory' ); ?>/static/img/external-icon.png" alt="" border="0" width="17" height="14">
```

Comment:
We should probably include the external icon image in the above link so that it's clickable as well. We could just put it in right next to 'Announcements' with no space, and add a padding-left to the image. Otherwise, the underline shows on the space in between and it looks funky :)